projects/addon-doc/src/components/language-switcher/language-switcher.module.ts

```diff
@@ -0,0 +1,22 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
+import {TuiDataListModule, TuiTextfieldControllerModule} from '@taiga-ui/core';
+import {TuiDataListWrapperModule, TuiSelectModule} from '@taiga-ui/kit';
+
+import {TuiLanguageSwitcherComponent} from './language-switcher.component';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        FormsModule,
+        TuiSelectModule,
+        TuiDataListModule,
+        ReactiveFormsModule,
+        TuiDataListWrapperModule,
+        TuiTextfieldControllerModule,
```

Comment:
My last comment still stands. `FormsModule`, `TuiDataListWrapperModule` and `TuiTextfieldControllerModule` are not needed.

Reply:
fixed